views/blog.erb

```diff
@@ -0,0 +1,61 @@
+<html>
+  <head>
+    <title>Lauren Granger's Awesome Website of Awesomeness</title>
+  </head>
+
+  <header>
+    <h1>Welcome to Lauren's Awesome Website of Awesomeness</h1>
+  <nav>
+    <ul>
+      <strong><li><a href="index">Home</a></li></strong>
+      <li><a href="about_me">All About Lauren</a></li>
+      <li><a href="blog">Blog</a></li>
+      <li><a href="projects">Projects</a></li>
+    </ul
+  </nav>
+  <img src="./images/break_line.png"/>
+  </header>
+
+  <table>
+    <thead>
+      <th colspan="2">
+        <tr>
+          3 lbs of Thought
+        </tr>
+        <tr><i>
```

Comment:
It looks like your nesting is incorrect here, you open an `i`, close a `tr` and then close the `i`